src/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -106,6 +179,25 @@ public static JsonDocument Parse(Stream utf8Json, JsonReaderOptions readerOption
             }
         }
 
+        /// <summary>
+        ///   Parse text representing a single JSON value into a JsonDocument.
```

Comment:
Should we specify the encoding?

Reply:
Here/elsewhere we have char overloads.

Reply:
Since it takes `char` (or `string`) there's no notion of encoding.  Encoding only applies to how raw bytes are interpreted.